MaxText/layers/linears.py

```diff
@@ -495,11 +495,29 @@ def load_balance_loss(self, top_k_indices, logits):
 
   def get_einsum(self, rhs_mesh_axes: Tuple[Optional[str], ...] = ()):
     if self.quant:
-      einsum_op = self.quant.einsum(rhs_mesh_axes)
+
+      def aqt_einsum(*args, **kwargs):
+        # aqt einsum doesn't support kwargs like precision
+        max_logging.log(f"[WARNING] {kwargs=} argument is not supported and will be ignored in aqt_einsum")
```

Comment:
I would remove this warning, it is just spammy IMO

Reply:
Sure!

Reply:
Removed.